src/graphql/statistics.rs

```diff
@@ -45,8 +44,6 @@ impl StatisticsQuery {
         last: Option<i32>,
     ) -> Result<Connection<String, Round, TotalCountByCluster, EmptyFields, RoundByCluster>> {
         let cluster = cluster.as_str().parse()?;
-        let (after, before, first, last) =
-            validate_and_process_pagination_params(after, before, first, last)?;
```

Comment:
It seems like the validation for the params is not handled afterwards for this endpoint.

Reply:
Later in the function it called `load_rounds_by_cluster` which includes:
``` Rust
  let after = slicing::decode_cursor(after)?.map(|(_, t)| i64_to_naive_date_time(t));
  let before = slicing::decode_cursor(before)?.map(|(_, t)| i64_to_naive_date_time(t));
```
The removal here is actually to fix the double decoding issue in the original code, since `validate_and_process_pagination_params` also decodes `after` and `before`. 
In addition, if `validate_and_process_pagination_params` is preferred for PostgresQL related parameter validation, then please create an issue and I'll attend to it in a separate PR.

Reply:
Thank you for clarifying it. I think input parameter validation is something to be run before the logic proceeds to do DB related `load_*` functions, so let me create another issue for this.